resources/stylesheets/base/_register.scss

```diff
@@ -0,0 +1,105 @@
+*{
+    padding: 0;
+    margin: 0;
+}

```

Comment:
this are called a reset styles. because it reset all the elements, so instead of doing in this file make a _reset.scss or use a _base.scss file inside the base folder to set the reset styles